docs/content/config/environment.md

```diff
@@ -4,7 +4,7 @@ title: Environment Variables
 
 !!! info
 
-    Values in **bold** are the default values. If an option doesn't work as documented here, check if you are running the latest image. The current `master` branch corresponds to the image `mailserver/docker-mailserver:edge`.
+    Values in **bold** are the default values. If an option doesn't work as documented here, check if you are running the latest image. The current `master` branch corresponds to the image `ghcr.io/docker-mailserver/docker-mailserver:edge`.
```

Comment:
Done in cbf06b8389bb1997d86871e9c28cefcaabd23f08. I consistently renamed nearly parts of our docs, but left out content related to `setup.sh`, because I will propose a dedicated PR to change this in `setup.sh` as well, which does not belong in this docs-update-PR.